app-doc/gimp-help/gimp-help-2.10.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-any-r1
+
+DESCRIPTION="GNU Image Manipulation Program help files"
+HOMEPAGE="https://docs.gimp.org/"
+SRC_URI="mirror://gimp/help/${P}.tar.bz2"
+
+LICENSE="FDL-1.2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+DEPEND="${PYTHON_DEPS}
+	$(python_gen_any_dep 'dev-libs/libxml2[python,${PYTHON_USEDEP}]')
+	dev-libs/libxslt
+	sys-devel/gettext
+"
+
+src_configure() {
+	econf --without-gimp
+}
+
+src_compile() {
+	# see https://bugs.gentoo.org/677198
+	emake -j 1
```

Comment:
Could you test if this is still the case? Does `-j 1` work, I've only seen `-j1`?

Reply:
My cpu has only 2 cores with HT (total 4 logical cores) so I maybe just coudn;t be affected this issue. I tried to use -j4 and -j20 without described error.

Fedora spec file still use disabled parallel build but maybe it was just copied.

I could remove this if it's necessary.